docs/anomaly-detection/components/models.md

```diff
@@ -617,7 +617,7 @@ Here in this guide, we will
 > **Note**: By default, each custom model is created as [**univariate**](#univariate-models) / [**non-rolling**](#non-rolling-models) model. If you want to override this behavior, define models inherited from `RollingModel` (to get a rolling model), or having `is_multivariate` class arg set to `True` (please refer to the code example below).
 
 We'll create `custom_model.py` file with `CustomModel` class that will inherit from `vmanomaly`'s `Model` base class.
-In the `CustomModel` class there should be three required methods - `__init__`, `fit` and `infer`:
+In the `CustomModel` class there should be three required methods - `__init__`, `fit`, `infer`, `serialize` and `deserialize`:
```

Comment:
Now it's more than three methods, we can rephrase using "several" or "a few" methods